1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -121,23 +121,19 @@ Path to a file in OpenSSL PEM format containing the complete CA certification ch
 The dictionary of Docker credentials to pass.
 
 - If unset, a default empty credentials file is created at `/etc/mesosphere/docker_credentials` during DC/OS install. A sysadmin can change credentials as needed. A `systemctl restart dcos-mesos-slave` or `systemctl restart dcos-mesos-slave-public` is required for changes to take effect.
```

Comment:
The current text seems to indicate that if we don't set anything, the default behavior is that this feature will be enabled with an empty credential file. But AFAIK this is true only if `cluster_docker_credentials_enabled` is set to true. In other words, all docker credential configs described in this section won't have any effect as long as `cluster_docker_credentials_enabled` is unset.

Reply:
@chhsia0  Please see amended note and separate cluster_docker_credentials_enabled description.